JavaScript/packages/recognizers-date-time/src/dateTime/chinese/dateConfiguration.ts

```diff
@@ -27,19 +27,22 @@ class ChineseDateExtractorConfiguration implements IDateExtractorConfiguration {
     readonly durationExtractor: BaseDurationExtractor;
     readonly utilityConfiguration: IDateTimeUtilityConfiguration;
 
-    constructor() {
+    constructor(dmyDateFormat: boolean = false) {
```

Comment:
I recommend removing the default value to make sure the top-level configuration pass down.

Reply:
The same to other low-level signature.